codalab/worker/main.py

```diff
@@ -116,6 +117,11 @@ def parse_args():
         action='store_true',
         help='If specified the worker quits if it finds itself with no jobs after a checkin',
     )
+    parser.add_argument(
+        '--container_runtime',
```

Comment:
Test running this with docker / singularity to make sure it works.

Reply:
will do